pkg/apis/extensions/install/install.go

```diff
@@ -56,6 +54,15 @@ func init() {
 		return
 	}
 
+	registered.RegisterVersions(availableVersions...)
+
+	if err := registered.EnableVersions(externalVersions...); err != nil {
+		glog.V(4).Infof("%v", err)
+		return
+	}
+
+	addVersionsToScheme(externalVersions...)
+
 	preferredExternalVersion := externalVersions[0]
 
 	groupMeta := latest.GroupMeta{
```

Comment:
As a separate issue (out of scope of this PR, I believe), all this code to enable the version should not be present here.
This code should belong with api machinery code and not with specific api code.
Someone writing a package for a new API group version should just call some genericApiMachinery.EnableVersion() and pass it the required params. the generic api machinery code should know that it needs to call latest.RegisterGroup and api.RegisterRESTMapper, not each api group version package.